portfolio/src/main/webapp/index.html

```diff
@@ -91,6 +91,15 @@
       <button class="form-button" type="submit" name="button">Send</button>
     </form>
 
+    <div id="images-container">
+      <form id="image-upload-form" class="hidden" method="POST" enctype="multipart/form-data">
+        <h1>Upload an image:</h1>
+        <input type="file" name="image">
+        <br/><br/>
+        <button>Submit</button>
```

Comment:
This button doesn't have a `type="submit"`

Reply:
done